https_mirror_install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+cd $(mktemp -d)
+curl -o sopstool.tar.gz https://oss-pkg.ibotta.com/sopstool/sopstool_linux.tar.gz
+tar -xvzf sopstool.tar.gz
+curl -o sops.tar.gz https://oss-pkg.ibotta.com/sops/sops_linux.tar.gz
+tar -xvzf sops.tar.gz
+mv sops /bin
+mv sopstool /bin
```

Comment:
/bin is not a great place to put things, we should put it within /usr/local/bin. We can use the `install` tool to copy with the appropriate rights, etc.

We should look into seeing if we can augment the existing installer scripts to help with some of these commands, as they're pretty shared.

Reply:
👍 I agree, with `/bin` (I'll make it default to `/user/local/bin` and give the option to input your own location) and it would be nice if we got rid of redundancy with the existing installers. 

I'll make some changes.